spec.yaml

```diff
@@ -1088,6 +1088,85 @@ paths:
           schema:
             $ref: "./definitions.yaml#/definitions/V4GenericResponse"
 
+  /v4/clusters/{cluster_id}/apps/:
+    get:
+      operationId: getClusterApps
+      tags:
+        - app katalog
+        - clusters
```

Comment:
this means it shows up both in the `Clusters` and `App Katalog` tab in the api docs.

Reply:
hmm but it is only returning installed apps on that cluster, or the list of all apps? (installed and available to install)

Reply:
Only the apps installed on that cluster.

Reply:
then I am not sure it will be correct list the endpoint on the app katalog. I see it can get mixed with the list available apps, dont you think?

Reply:
hmm.. would like some other feedback on that. To me its pretty clear.

list of available apps is the helm repo index at the `url` you get for a catalog when doing 

`GET /v4/appcatalogs/`  (different PR)

and then the list of apps on a cluster is

`GET /v4/clusters/:clusterid/apps/`  (this pr)

Reply:
yeah, I think Fer is wondering if listing this under catalog would make people think there /ve/appcatalogs/:catalog/apps